Core/CMakeLists.txt

```diff
@@ -216,20 +218,22 @@ target_link_libraries(elastix_lib ${ELASTIX_TARGET_LINK_LIBRARIES})
 #---------------------------------------------------------------------
 # Create the transformix executable.
 
-add_executable(transformix_exe
-  Main/transformix.cxx
-  Main/elastix.h
-  Main/elxMainExeUtilities.cxx
-  Main/elxMainExeUtilities.h
-  Kernel/elxMainBase.cxx
-  Kernel/elxMainBase.h
-  Kernel/elxTransformixMain.cxx
-  Kernel/elxTransformixMain.h
-  ${InstallFilesForExecutables}
-)
-set_target_properties(transformix_exe PROPERTIES OUTPUT_NAME transformix)
-target_compile_definitions(transformix_exe PRIVATE ELX_CMAKE_VERSION="${CMAKE_VERSION}")
-target_link_libraries(transformix_exe ${ELASTIX_TARGET_LINK_LIBRARIES})
+if(NOT WASI AND NOT EMSCRIPTEN)
```

Comment:
OK, but should this be `if(ELASTIX_EXECUTABLE_BUILDS)`?

Reply:
Update: The CMake option `ELASTIX_BUILD_EXECUTABLE` has just been added to the main branch! With pull request #922 commit 9d76b86f955043e2ae17bbbd621aaa5f0f60e001  @thewtex  Please rebase your wasm branch, and remove these two commits:

  - COMP: Do not build elastix and transformix exe with Wasm
  - COMP: Provide default ELX_CMAKE_VERSION

"COMP: Provide default ELX_CMAKE_VERSION" should no longer be necessary, because `ELX_CMAKE_VERSION` was only used when building the executables.

Reply:
Done